frontend/src/data/audit-events.json

```diff
@@ -32,6 +32,8 @@
         "project.stopped": "Project Stopped",
         "project.restarted": "Project Restarted",
         "project.suspended": "Project Suspended",
+        "project.copied": "Project Copied",
+        "project.imported": "Project Imported",
```

Comment:
```suggestion
        "project.promoted_deploy": "Project Deployed from XYZ",
```

Reply:
not possible to have injections in here, need to be static text

Reply:
I am happy to change the terminology to "Project Promoted" - but it does leave the open issue that this will also be used when users interact with "Copy Project"

Reply:
For what it's worth, users are using the "Copy" Project. 22 users on FF Cloud have used it in the past 3 months. Nobody has used the "Export to Existing Project" button.

Reply:
I'd be fine with removing the "Export" feature or button. Let me know what you think

Reply:
I think this feature largely replaces the purpose of the Export button. No need to have both, so lets remove the Export button.